examples/genrule/BUILD.bazel

```diff
@@ -67,7 +67,9 @@ genrule(
     name = "require_acorn",
     srcs = [
         ":require_acorn_js",
+        # reference the location where the "acorn" npm package was linked in our root Bazel package.
         "//:node_modules/acorn",
+        "//:node_modules/acorn/dir",
```

Comment:
some users will see this breaking change too? need to document in the commit?

Reply:
👍  will mention that.